php.ini-production

```diff
@@ -1764,20 +1764,20 @@ ldap.max_links = -1
 
 [opcache]
 ; Determines if Zend OPCache is enabled
-;opcache.enable=0
+;opcache.enable=1
 
 ; Determines if Zend OPCache is enabled for the CLI version of PHP
-;opcache.enable_cli=0
+;opcache.enable_cli=1
```

Comment:
SInce it's commented out and will have to be manually enabled by the admin anyway, does it make much difference?